### DISCUSSION TITLE
How to disable the Bézier smooth between points for chart?

### DISCUSSION THREAD

**Discussion by zch1234qq** [patrykandpatrick/vico#338] [Questions]

**Comment by patrickmichalik** [comment]
Hello! Setting `DefaultPointConnector#cubicStrength` to zero produces this behavior. Please see the fourth sample chart.

**Reply by zch1234qq** [reply]
thanks bro!

**Reply by patrickmichalik** [reply]
You’re welcome!

**Reply by JayantSharma777** [reply]
Hi,Can you also give the solution for ChartView for the same problem?

**Reply by patrickmichalik** [reply]
The solution is the same, @JayantSharma777. You can use the `chart` field of `ChartView` to access your `LineChart` instance. Please take a look at the fourth sample chart.

**Reply by JayantSharma777** [reply]
Yup thanks, got it right, had to make custom styles in the xml to make it work.
Also, one more thing,I want that in the line graph, at every particular point, i want to add a drawable describing that point(like a marker but for every point). Can this be achieved?
 Thanks for making this library, great customizations are available.

**Reply by patrickmichalik** [reply]
Thanks for the kind words! Yes, this is possible. Please take a look at the `pointStyle` and `pointSize` attributes of the `LineSpec` attribute set. A full list of attributes can be found [here](https://patrykandpatrick.com/vico/wiki/top-level-elements/#xml-attributes).

**Reply by JayantSharma777** [reply]
I tried it but, somehow the pointStyle is of type ComponentStyle and there also the shapeStyle is of Shape type,I am not able to find an attribute to add my custom drawable.

**Reply by patrickmichalik** [reply]
Ah, I apologize. If you’d like to use a `Drawable`, please see [`drawableShape`](https://patrykandpatrick.com/vico/api/core/com.patrykandpatrick.vico.core.component.shape/-shapes/drawable-shape).

**Reply by JayantSharma777** [reply]
Yup I can look for the drawable shape but however I can not use drawableShape in my styles.xml to change my PointShape
<img width="538" alt="image" src="https://github.com/patrykandpatrick/vico/assets/75779183/dff4b5b9-cdb7-445b-bdc5-0a9a1a2d76c5">
Here is my styles.xml file,please let me know how and where can I add my drawable in it.
Thanks

**Reply by patrickmichalik** [reply]
There aren’t XML attributes for `drawableShape` at the moment, @JayantSharma777. You’ll have to use the function directly (i.e., from Kotlin). We’ll be looking into adding XML attributes for `drawableShape` down the line.

**Reply by JayantSharma777** [reply]
Ok thanks @patrickmichalik

**Reply by patrickmichalik** [reply]
You’re welcome!

**Comment by zch1234qq** [comment]
thanks vico
